MapboxNavigation/RouteTableViewController.swift

```diff
@@ -61,7 +60,7 @@ class RouteTableViewController: UIViewController {
             guard coordinatesLeftOnStepCount <= congestionTimesForStep.count else { return }
             
             let remainingCongestionTimesForStep = congestionTimesForStep.suffix(from: coordinatesLeftOnStepCount)
-            let remainingCongestionTimesForRoute = routeProgress.congestionTimesPerStep[routeProgress.legIndex][routeProgress.currentLegProgress.stepIndex + 1..<routeProgress.currentLeg.steps.count]
+            let remainingCongestionTimesForRoute = routeProgress.congestionTimesPerStep[routeProgress.legIndex].suffix(from: routeProgress.currentLegProgress.stepIndex + 2)
```

Comment:
This will crash if `congestionTimesPerStep[routeProgress.legIndex]` has only two items and we’re on step 1.